src/app/containers/Article/index.jsx

```diff
@@ -36,9 +36,14 @@ class ArticleContainer extends Component {
   render() {
     const { data } = this.props;
     const { content, metadata, promo } = data;
+    const { id: aresArticleId } = metadata;
+
+    const idParts = aresArticleId.split(':');
```

Comment:
It would be less verbose to just do `aresArticleId.split(':').pop()` to get the last item in the array.